templates/word-daily-level/src/addons/words/grid-style.css

```diff
@@ -0,0 +1,47 @@
+word-game-grid {
+    display: block;
+}
+
+/* Grid */
+word-game-grid .cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    font-weight: bold;
+    color: black;
+    background-color: white;
+    border: 2px solid black;
+}
+
+/* Grid row */
+word-game-grid > div {
+    display: flex;
+    justify-content: center;
+    gap: 2px;
+    margin: 2px;
+}
+
+word-game-grid .cell[s=b] {
```

Comment:
`.cell[s=b]` could be more intuitive. Maybe `.cell-state-bull`?

Reply:
As an aside should we use the cows/bulls terminology or come up with something else?